src/docs/common_tasks/file_bundles.md

```diff
@@ -13,15 +13,16 @@ Add a `bundles` list to your `jvm_app` target definition (more info on `jvm_app`
 Here is an example `jvm_app` definition that shows different possibilities for defining file bundles (and explains them below):
 
     ::python
-    jvm_app(name='bundle',
-      bundle(fileset=rglobs('src/main/resources/*')), # 1
-      bundle(fileset=globs('config/*') - globs('*.xml'), # 2
-      bundle(relative_to='src', fileset=globs('src/scripts/*.sh')) # 3
+    jvm_app(
+      name='bundle',
+      bundle(fileset=['src/main/resources/**/*')), # 1
+      bundle(fileset=['config/*', '!*.xml']), # 2
+      bundle(fileset=['src/scripts/*.sh'], relative_to='src') # 3
```

Comment:
Side note: once we finish deprecating `globs` et al., IMO we should update `bundle` to no longer use `fileset` as a kwarg - we always require the field so it's a bit redundant.

Reply:
Should this example be in triple back ticks? Github is making all this text bold/italic for me, and as this is a markdown file it should probably render more tidily.

Reply:
oh, is `::python` supposed to handle that? Maybe this is github being odd.